examples/components/CCFRAUD_VERTICAL/traininsilo/host_spec.yaml

```diff
@@ -28,6 +28,18 @@ inputs:
     type: uri_folder
     description: a given pre-existing checkpoint
     optional: true
+  contributor_1_embeddings:
```

Comment:
I'm not a fan of the idea of having a fixed number of inputs, which depends on the number of silos. Is there any other way ?

Could we do a first scatter/gather phase just to generate embeddings in an aggregated folder, then do the training in silos?

Reply:
This is the same issue as with `aggregatemodelweights` component and I believe we have not figured out any other better way for now, right?

Reply:
Neither me but I believe we do no have better solution, otherwise we would already implement it for `aggregatemodelweights` right?